microprofile.ls/README.md

```diff
@@ -43,8 +46,52 @@ Building the Language Server
 Clients
 -------
 
-Here are some clients consuming the Quarkus language server:
+Here are some clients consuming the MicroProfile language server:
 
  * Eclipse with [quarkus-lsp4e (POC)](https://github.com/angelozerr/quarkus-lsp4e)
  * IntelliJ with [intellij-quarkus](https://github.com/jeffmaury/intellij-quarkus)
- * Visual Studio Code with [vscode-quarkus](https://github.com/redhat-developer/vscode-quarkus)
\ No newline at end of file
+ * Visual Studio Code with [vscode-quarkus](https://github.com/redhat-developer/vscode-quarkus)
+ 
+Code Snippets
+-------
+
+Java and properties completion snippets are managed by the MicroProfile LS (snippets on server side) by using Java SPI.
+
+## Describes snippets in JSON
+
+Snippets are described in a JSON files by following the [vscode snippet format](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_create-your-own-snippets).
+
+A context can be described to show the snippet according to a condition:
```

Comment:
The `"context"` key can be provided to specify the condition in which the snippet should appear: